src/panel.css

```diff
@@ -435,14 +435,6 @@ h3.sites-allowed {
   background-image: url("/img/favi-messenger.png");
 }
 
-.favi-whatsapp {
-  background-image: url("/img/favi-whatsapp.png");
-}
-
-.favi-workplace {
-  background-image: url("/img/favi-workplace.png");
-}
-
```

Comment:
This feels like a artifact from a bad `rebase`. These file changes should be reverted.

Reply:
Outdated.